classes/hazelcast-local-config.template.xml

```diff
@@ -2,35 +2,36 @@
 <hazelcast xmlns="http://www.hazelcast.com/schema/config"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://www.hazelcast.com/schema/config
-           http://www.hazelcast.com/schema/config/hazelcast-config-3.12.xsd">
-    <group>
-        <name>openfire</name>
-        <password>openfire</password>
-    </group>
+           https://www.hazelcast.com/schema/config/hazelcast-config-5.3.xsd">
+    <!-- Recommend you open cp subsystem on your cluster -->
+    <cp-subsystem>
+        <cp-member-count>3</cp-member-count>
+        <group-size>3</group-size>
+    </cp-subsystem>
     <network>
         <port auto-increment="true" port-count="100">5701</port>
         <outbound-ports>
             <ports>0</ports>
         </outbound-ports>
-        <!-- The following enables multicast discovery of cluster members
-                See http://docs.hazelcast.org/docs/3.12/manual/html-single/index.html#discovering-members-by-multicast
-        -->
         <join>
+            <!-- More about join type, you can get help from
+            https://github.com/hazelcast/hazelcast/blob/v5.3.7/hazelcast/src/main/resources/hazelcast-full-example.xml
+            Now support multicast, tcp-ip, aws, gcp, azure, kubernetes, eureka, and your self discovery-strategies -->
             <multicast enabled="true">
                 <multicast-group>224.2.2.3</multicast-group>
                 <multicast-port>54327</multicast-port>
             </multicast>
-            <tcp-ip enabled="false"/>
         </join>
         <!-- The following enables TCP/IP based discovery of cluster members
-                See http://docs.hazelcast.org/docs/3.12/manual/html-single/index.html#discovering-members-by-tcp
+        See https://github.com/hazelcast/hazelcast/blob/v5.3.7/hazelcast/src/main/resources/hazelcast-full-example.xml#L490
         -->
         <!--
         <join>
             <multicast enabled="false"/>
             <tcp-ip enabled="true">
                 <member>10.10.1.1:5701</member>
-				<member>10.10.1.2:5701</member>
+                <member>10.10.1.2:5701</member>
+               </member-list>
```

Comment:
This looks like you're closing tag that AFAICT isn't opened anywhere. Having read the doc's I don't think it's needed.